partner_firstname/views/res_partner.xml

```diff
@@ -0,0 +1,79 @@
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
```

Comment:
Why did you delete the old extension of view_partner_simple_form? I think it's still needed as well as this other extension

Reply:
You are right. I forgot to add this back again to the PR. Same problem here though, as with the view_partner_form. Read more below and please suggest a solution for this.